### ISSUE TITLE
feat:  support configure multi plugin config in route

### ISSUE THREAD

**Issue by zsmlinux** [apache/apisix#8401]
### Description

Now when configure more one plugin config in route，only one takes effect

**Comment by Hazel6869** [comment]
@zsmlinux yes  one plugin config has effect for one plugin, you can use multi-routing.

**Comment by shreemaan-abhishek** [comment]
Why do you need multiple plugin configs in one route? Why not just create another plugin config that contains all the plugins you need and use it in the route? Or just add all the plugins you need in the route directly.

**Comment by zsmlinux** [comment]
For instance, there's a plugin config A that is used by Route A and B. And Route A expects to add other plugin config while Route B doesn't. So, I hope to add another plugin config B to Route A. At the same time, if the route supports configuring multiple plugin configs, it can be very effective to use plugins in combination.

**Comment by shreemaan-abhishek** [comment]
would you like to contribute a PR for this?

**Comment by Baoyuantop** [comment]
Closed due to prolonged lack of progress. If you're still interested in this feature, please let us know.
